pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -231,6 +245,9 @@ func buildFileAccessLogHelper(path string, mesh *meshconfig.MeshConfig) *accessl
 				jsonLogStruct = &parsedJSONLogStruct
 			}
 		}
+		if _, exists := jsonLogStruct.Fields[requestWithoutQuery]; exists {
```

Comment:
wouldn't requestWithoutQuery be in the values, not the keys?

Reply:
Oh. I thought is field name with empty value. I have changed. PTAL